main.js

```diff
@@ -18,7 +18,7 @@ function promisify(func) {
       args.push(function (err) {
         if (err !== null) { return reject(err); }
         var rest = Array.prototype.slice.call(arguments, 1);
-        return resolve(rest.length === 1 ? rest[0] : rest);
+        return rest.length === 1 ? resolve(rest[0]) : resolve();
```

Comment:
This may drop information for functions with more than one parameter in their callback.

```js
if (rest.length === 0) {
  return resolve();
} else {
  return resolve(rest.length === 1 ? resolve(rest[0]) : rest);
}
```

Or maybe something like

```js
return resolve(rest.length > 1 ? resolve(rest) : rest.length === 1 ? rest[0] : undefined);
```

for style points.